clicommand/annotate.go

```diff
@@ -109,6 +116,11 @@ var AnnotateCommand = cli.Command{
 		ctx, cfg, l, _, done := setupLoggerAndConfig[AnnotateConfig](ctx, c)
 		defer done()
 
+		validScopes := []string{"build", "job"}
```

Comment:
We may add a step scope in the future - could be good to leave this to server side validation to make that easier?